src/WebJobs.Script/Binding/ApiHubTableBinding.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.ObjectModel;
+using System.IO;
+using System.Reflection.Emit;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Host.Bindings.Runtime;
+using Microsoft.Azure.WebJobs.Script.Description;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Script.Binding
+{
+    public class ApiHubTableBinding : FunctionBinding
+    {
+        public ApiHubTableBinding(
+            ScriptHostConfiguration config, 
+            ApiHubTableBindingMetadata metadata, 
+            FileAccess access) 
+            : base(config, metadata, access)
+        {
+            Connection = metadata.Connection;
+            DataSetName = metadata.DataSetName;
+            TableName = metadata.TableName;
+            EntityId = metadata.EntityId;
+            BindingDirection = metadata.Direction;
+        }
+
+        public string Connection { get; }
+
+        public string DataSetName { get; }
+
+        public string TableName { get; }
+
+        public string EntityId { get; }
+
+        public BindingDirection BindingDirection { get; }
+
+        public override Collection<CustomAttributeBuilder> GetCustomAttributes(Type parameterType)
+        {
+            var constructorTypes = new[] { typeof(string) };
+            var constructor = typeof(ApiHubTableAttribute).GetConstructor(constructorTypes);
+            var constructorArguments = new[] { Connection };
+            var namedProperties = new[]
+            {
+                typeof(ApiHubTableAttribute).GetProperty("DataSetName"),
+                typeof(ApiHubTableAttribute).GetProperty("TableName"),
+                typeof(ApiHubTableAttribute).GetProperty("EntityId")
+            };
+            var propertyValues = new[]
+            {
+                DataSetName,
+                TableName,
+                EntityId
+            };
+
+            return new Collection<CustomAttributeBuilder>()
+            {
+                new CustomAttributeBuilder(
+                    constructor,
+                    constructorArguments,
+                    namedProperties,
+                    propertyValues)
+            };
+        }
+
+        public override async Task BindAsync(BindingContext context)
+        {
+            var attribute = new ApiHubTableAttribute(Resolve(Connection))
+            {
+                DataSetName = Resolve(DataSetName),
```

Comment:
The Resolve call only applies NameResolver parameters, not binding data coming from the request. You really want to support those scenarios (e.g. allowing your EntityId property to include an {Id} that resolves from the incoming queue message,etc.). See how QueueBinding and the other bindings use BindingTemplates for this.